File: planner-todo/src/main/java/ru/serdyuk/micro/planner/todo/service/StatService.java
package ru.serdyuk.micro.planner.todo.service;

import org.springframework.stereotype.Service;
import ru.serdyuk.micro.planner.entity.Stat;
import ru.serdyuk.micro.planner.todo.repo.StatRepository;

import javax.transaction.Transactional;

@Service
@Transactional
public class StatService {

    private final StatRepository repository;

    public StatService(StatRepository repository) {
        this.repository = repository;
    }

    public Stat findStat(Long userId) {
        return repository.findByUserId(userId);
    }
}

File: planner-todo/src/main/java/ru/serdyuk/micro/planner/todo/service/StatService.kt
package ru.serdyuk.micro.planner.todo.service

import org.springframework.stereotype.Service
import ru.serdyuk.micro.planner.entity.Stat
import ru.serdyuk.micro.planner.todo.repo.StatRepository
import javax.transaction.Transactional

@Service
@Transactional
class StatService(private val repository: StatRepository) {
    fun findStat(userId: Long): Stat {
        return repository.findByUserId(userId)
    }
}